region/tourian/main/Tourian Escape Room 4.json

```diff
@@ -431,6 +437,12 @@
       ],
       "clearsObstacles": ["A"],
       "devNote": "IBJ can be used to avoid wall jumps before the acid starts rising, but a crouch jump down grab is needed near the right door."
+    },
+    {
+      "link": [3, 3],
+      "name": "Crystal Flash",
+      "requires": ["h_canCrystalFlash"],
+      "devNote": "The Crystal Flash should be performed on a platform away from the acid trigger or any Pirates."
```

Comment:
This should work.  CF before triggering the acid and entering the box that is Node 3.
[3, 2] adds a trivial movement requirement whereas [2, 2] would slowly kill the pirates.